Gems/AWSClientAuth/Code/CMakeLists.txt

```diff
@@ -69,58 +78,58 @@ ly_add_target(
             AZ::AzToolsFramework
             Gem::AWSCore
             Gem::HttpRequestor
-            3rdParty::AWSNativeSDK::AWSClientAuth
+            3rdParty::AWSNativeSDK::${gem_name}
```

Comment:
I don't think we want to use $gem_name here. If this gem name changes, the aws sdk will still be called AWSClientAuth.

Reply:
You are right, I careful not to change `3rdParty::AWSNativeSDK::AWSCore`, but I missed `3rdParty::AWSNativeSDK::AWSClientAuth`